src/myparcelcom.js

```diff
@@ -102,9 +102,6 @@
           'ShipmentSurcharges',
           'SystemMessages',
           'ExportLoginAttempts',
-          "Returns",
-          "ReturnMethods",
-          "ReturnReasons",
           "ReturnReasonsManagement",
```

Comment:
FYI I decided to keep the ReturnReasonsManagement hidden since it's not interesting for users.